lib/OpusVL/FB11/Role/Hat/dbicdh/consumer.pm

```diff
@@ -47,7 +47,7 @@ We always upgrade to the current version according to C<schema_version>.
 
 requires 'schema';
 
-sub priority {1}
+sub priority {10}
```

Comment:
I wonder whether `sequence` might actually be better.  Priority (without reading the POD) reads to me as "higher the sooner".

Reply:
Oof, yeah, priority did seem to be a pesky word when I used it but I couldn't come up with a situation where it was misconstrued. But I think my brain short-circuited it. A higher priority is more important, but the first priority is the highest one... yikes